.azure-pipelines/templates/variables.yml

```diff
@@ -1,6 +1,6 @@
 variables:
-  ubuntu_pool: 'pool-ubuntu-2004'
-  ubuntu_multi_core_pool: 'pool-ubuntu-2004-multi-core'
+  ubuntu_pool: 'pool-ubuntu-2204'
```

Comment:
The reason for not using `pool-ubuntu-latest` is that Azure PowerShell team has already created `pool-ubuntu-2204`.

Reply:
What's the ubuntu version in `pool-ubuntu-latest-multi-core`, 22.04 or 24.04?

Reply:
22.04，because 2204 is the latest one in 1ES image.